Dockerfile

```diff
@@ -25,7 +26,8 @@ RUN rm -f /etc/apt/sources.list.d/chris-lea-node_js-jessie.list \
 && curl https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
 && echo 'deb https://deb.nodesource.com/node_6.x jessie main' > /etc/apt/sources.list.d/nodesource.list \
 && apt-get update \
-&& apt-get install -y nodejs
+&& apt-get install -y nodejs \
+&& apt-get install -y libfontconfig
```

Comment:
This isn't a part of the node installation, so I think it should be its own `RUN` command, included in a more related command, or added to the list at  the top of the file.